### DISCUSSION TITLE
Podman REST API to set cgroup-manager to cgroupfs?

### DISCUSSION THREAD

**Discussion by jay-fyi** [containers/podman#10127] [Q&A]
Is there a way to use the Podman REST API to set the  `--cgroup-manager=cgroupfs` instead of in the `/usr/share/containers/containers.conf` config file?

**Comment by mheon** [comment]
You are looking for the `cgroup_manager` option - set it to `cgroupfs`.

**Reply by jay-fyi** [reply]
yes, that is what I am looking for where can I set that option in the Podman REST API? if that is possible and where would that be? any example would be appreciated thanks. 

I am using goLang and couldn't find a variable that sounds like it in the `specgen.go` fie.

**Reply by mheon** [reply]
You need to set it on the API service - you can't set it directly via the API but can set the server to use it. You'd have to modify `podman.service` to launch the `podman system service` command with the `--cgroup-manager=cgroupfs` argument.

**Reply by jay-fyi** [reply]
I see, got it. thank you!
